src/bobbit/modules/help.py

```diff
@@ -26,7 +26,11 @@ async def help(bot, message, module_name=None):
     if not responses:
         responses = sorted([m.NAME for m in bot.modules if module_name in m.NAME or module_name == 'all'])
 
-    return message.with_body(', '.join(responses))
```

Comment:
Let's keep this message if something is found, but we can use the new one if nothing is found.

So move this return inside the `if`